tests/Resources/ODS.robot

```diff
@@ -338,3 +338,23 @@ OpenShift Resource Component Field Should Not Be Empty
     [Arguments]    ${resource_component_field}
     Run Keyword And Continue On Failure    Should Not Be Empty    ${resource_component_field}
 
+
+Reboot Cluster Node
+    [Documentation]    Reboots the specified node of the cluster
+    ...    Args:
+    ...        node(str): Name of the node to reboot
+    ...    Returns:
+    ...        None
+    [Arguments]    ${node}
+    Run    oc debug node/"${node}" -T -- sh -c "echo b > /proc/sysrq-trigger"
+
+Verify ODS Availability
+    [Documentation]    Verifies that there is no downtime in ODS
+    ...    Returns:
+    ...        None
+    ${expression}=    Set Variable    rhods_aggregate_availability[10m : 1m]
+    ${resp}=    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
+    @{values}=    Set Variable    ${resp.json()["data"]["result"][0]["values"]}
+    FOR    ${value}    IN    @{values}
+        Should Not Be Equal As Strings    ${value}[1]    0
+    END
```

Comment:
maybe this one could fit more in the resources files about metrics. (I'm okay with the current solution too)

Reply:
done